honeypot/Datadog.Dependabot.Integrations.Honeypot.csproj

```diff
@@ -0,0 +1,201 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <RootNamespace>Datadog.Dependabot.Honeypot</RootNamespace>
+  </PropertyGroup>
```

Comment:
Does this csproj need to be at the root level? If not, I wonder if we should move it into the _tracer_ subdirectory, as it's currently tracer specific isn't it?